src/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 
 option(USE_FTGL "build with ftgl" ON)
 option(USE_JACK "build with Jack (Only required of BSD Unix)" OFF)
+option(USE_SYSTEM_RTMIDI "build with system (not bundled) RtMidi" OFF)
```

Comment:
Please can you default this to ON. As I want to reduce the amount of testing that is required. 

Actually I have an even better idea. I would be most grateful if you could change this two USE_BUNDLED_RTMIDI and set that to default to OFF. I would really like both the cmake .. and  distributions to use the same build options where possible. Otherwise I have to ask users every time about this switch when trying to recreate their problem.

Reply:
The vast majority of distributions will not use a bundled lib but the system one.

Reply:
Please invert the option and call it `USE_BUNDLED_RTMIDI` 

so that line might read
```
option(USE_BUNDLED_RTMIDI "build with bundled rtmidi  (for older distributions only)" OFF)
```